my-site.rb

```diff
@@ -2,7 +2,23 @@
 
 class MySite < Sinatra::Base
 
-  get "/" do
```

Comment:
It is good to leave the original "root route" here to ensure that the user has  a "default" place to go